src/datasets/pings.md

```diff
@@ -1,19 +1,133 @@
 # Raw Ping Data
 
+> **⚠** This article discusses pings sent by Firefox's legacy telemetry system.
```

Comment:
```suggestion
> **⚠** This article discusses pings sent by Firefox's legacy v4 telemetry system.
```

Firefox has had a few legacy telemetry systems; this is the one we still use. :D